frontend/test/__support__/e2e/config.js

```diff
@@ -72,12 +70,19 @@ const defaultConfig = {
         "frontend/test/snapshot-creators/qa-db.cy.snap.js";
     }
 
+    // `grepIntegrationFolder` needs to point to the root!
+    // See: https://github.com/cypress-io/cypress/issues/24452#issuecomment-1295377775
+    config.env.grepIntegrationFolder = "../../../../";
+    config.env.grepFilterSpecs = true;
+
     config.env.HAS_ENTERPRISE_TOKEN = hasEnterpriseToken;
     config.env.HAS_SNOWPLOW_MICRO = hasSnowplowMicro;
     config.env.SNOWPLOW_MICRO_URL = snowplowMicroUrl;
     config.env.SOURCE_VERSION = sourceVersion;
     config.env.TARGET_VERSION = targetVersion;
 
+    require("@cypress/grep/src/plugin")(config);
```

Comment:
The order matters.
All envs need to be ready for the grep plugin.

Reply:
Would it be good to include a comment about that in the file?